migrations/disable-erlang-support.js

```diff
@@ -5,9 +5,9 @@ module.exports = {
   created: new Date(2016, 10, 5),
   run: function(callback) {
     db.request({
-      db: '_config',
+      db: '_node',
```

Comment:
Arguably the correct thing to do would be to just use `request` the library and don't use nano for this at all. It's a migration, so I figured the least amount of change the better.